frontend/src/components/regions/CardDetails/Presentation.js

```diff
@@ -73,6 +84,9 @@ const CardBasicDetails = ({ card }) => {
           <FlavourChips flavourNames={card.flavourNames} />
           <StoryPoints storyPoints={card.storyPoints} />
         </Grid>
+        <TableCell>
+          <CardStatusChip card={card} />
+        </TableCell>
```

Comment:
It's a little odd to use a tablecell here. Where is the rest of the table?  Dont you want to use Grid things rather?